x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-config-topics.adoc

```diff
@@ -34,23 +32,22 @@ To configure schemas in Anypoint Studio:
 
-If you don't include any schemas in the configuration, the 12 EDI attempts to load an appropriate schema at runtime for each transaction set,
-using the standard X12 schema definitions provided with the connector. However, in this case you won't see any metadata to define the structure of the
+If you don't include any schemas in the configuration, the X12 EDI attempts to load an appropriate schema at runtime for each transaction set by using the standard X12 schema definitions provided with the connector. However, in this case you won't see any metadata to define the structure of the
 data passed in and out of the connector.
 
-If the schema definition configuration includes a 997 or 999 Functional Acknowledgment schema then that schema is used to generate Functional Acknowledgments.
+If the schema definition configuration includes a `997` or `999` Functional Acknowledgment schema then that schema is used to generate Functional Acknowledgments.
 
-NOTE: To use the Functional Acknowledgment schema type (997 or 999) it must match the `generate999Acks` Parser parameter (`false` for 997, true for `999`).
+NOTE: To use the Functional Acknowledgment schema type (`997` or `999`) it must match the `generate999Acks` Parser parameter (`false` for `997`, `true` for `999`).
 
 [%header%autowidth.spread]
 |===
 |XML Value |Studio Option
-|formValidation="X12"/"HIPAA_SNIP1"/"HIPAA_SNIP2" |The schema form and validation level can be either X12 or HIPAA. The default is X12.
-|schemas=list of values |List of schema paths the connector can use. These paths can be either a file system or `classpath`.
-|characterEncoding="PLATFORM"/"ASCII"/"ISO8859_1"/"IBM1047"/"UTF8" |The character encoding used for both send and receive messages. The default `PLATFORM` is the Mule platform default encoding.
-|stringCharacterSet="BASIC"/"EXTENDED"/"UNRESTRICTED" |Use a substitution character replacement for invalid characters, or reject them as errors if none is set or enabled for receive messages in the parser options.  The default is `EXTENDED` for string data.
-|stringSubstitutionChar="?" |Substitution character used to replace invalid characters in string values. The default is `none`.
-|versionIdentifierSuffix="XYZ" |Version identifier code suffix for the group (GS). Setting this value affects both send and receive message handling. On the receive side, the GS08 version code must contain the X12 schema version (for example, 004010, 005010, and so forth) followed by this suffix. Not setting this value means the receive-only just verifies that the GS version identifier code starts with the configured schema version. On the send side, the combination of the X12 schema version and this suffix comprises the GS08 value.
+|formValidation="X12"/"HIPAA_SNIP1"/"HIPAA_SNIP2" |The schema form and validation level can be either X12 or HIPAA. The default is X12
+|schemas=list of values |List of schema paths the connector can use. These paths can be either a file system or `classpath`
```

Comment:
You are missing a period in the sentence here and elsewhere.

Reply:
Fixed.